pkg/mount/mount.go

```diff
@@ -702,6 +784,16 @@ func New(
 	return nil, ErrUnsupported
 }
 
+// bindMountPath return the path used to bind mount to a user given mount path.
+// This allows us to chattr the path in case we cannot chattr the original mount path.
+func bindMountPath(path string) (string, error) {
```

Comment:
This will not work for shared volumes where the last item in the path can be the volumeID and we get two mount request on the same node.